torch/autograd/saved_variable_default_hooks.py

```diff
@@ -1,13 +1,85 @@
 import torch
-from typing import Any
+from typing import Callable, Any
 
-def set_saved_tensors_default_hooks(pack_hook, unpack_hook):
+def set_saved_tensors_default_hooks(pack_hook: Callable[[torch.Tensor], Any], unpack_hook: Callable[[Any], torch.Tensor]):
+    r"""Sets a pair of pack / unpack hooks for saved tensors.
+
+    Use these hooks to define how intermediary results of an operation
+    should be packed before saving, and unpacked on retrieval.
```

Comment:
Here is where I would add a link to https://github.com/pytorch/pytorch/pull/62362 which defines the terms `packing` and `unpacking`.